lib/mais_orcid_client.rb

```diff
@@ -6,6 +6,7 @@
 require 'faraday'
 require 'faraday/retry'
 require 'oauth2'
+require 'ostruct'
```

Comment:
i had to add this specifically as a dependency and require it to get it to work locally via `bin\console`